package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mercurius",
-  "version": "14.1.0",
+  "version": "15.0.0",
```

Comment:
Please revert, we'll do this ourselves. Out of question, why is this semver-major?

Reply:
I'm not sure, it happened automatically in this commit [6f18c6c](https://github.com/mercurius-js/mercurius/pull/1115/commits/6f18c6cb7553efb32e3e4f331b6ca5c64cd0fe09)

Reply:
Looks like a failed rebase – this PR is based on https://github.com/mercurius-js/mercurius/commit/fe4c63d28037ace32f0e9da69a698e5ce48454b0 but seems to be mistakenly squashed with the commit that follows it in the main repository